packages/kolibri-common/composables/useCoachMetadataTags.js

```diff
@@ -0,0 +1,73 @@
+import { ref } from 'kolibri.lib.vueCompositionApi';
+import { ContentNodeKinds } from 'kolibri.coreVue.vuex.constants';
+import coreStrings from 'kolibri.utils.coreStrings';
+import commonCoreStringsMixin from 'kolibri.coreVue.mixins.commonCoreStrings';
+
+export function useCoachMetadataTags(contentNode) {
+  const tags = ref([]);
+
+  const getCategoryTags = () => {
+    if (!contentNode.categories) return [];
+    return contentNode.categories.map(category =>
+      commonCoreStringsMixin.methods.coreString(category),
+    );
+  };
+
+  const getLevelTags = () => {
+    if (!contentNode.grade_levels) return [];
+    return contentNode.grade_levels.map(grade_levels =>
+      commonCoreStringsMixin.methods.coreString(grade_levels),
+    );
+  };
+
+  const getLanguageTag = () => {
+    if (!contentNode.lang) return [];
+    return contentNode.lang.length > 1 ? ['Multiple languages'] : [contentNode.lang.lang_name];
+  };
+
+  const getActivityTag = () => {
+    if (!contentNode.learning_activities) return [];
+
+    return contentNode.learning_activities.length > 1
+      ? ['Multiple learning activities']
+      : contentNode.learning_activities.map(activity =>
+          commonCoreStringsMixin.methods.coreString(activity),
+        );
+  };
+
+  const getDurationTag = () => {
+    if (!contentNode.duration) return [];
+    return contentNode.duration.map(duration => coreStrings.formatDuration(duration));
+  };
+
+  const getSpecificCategoryTag = () => {
```

Comment:
Just curious, was there a particular reason for using arrow functions here rather than defining it like `function getSpecificCategoryTag() {...}`?

Reply:
There's no real reason to use an arrow function here—it's just a style preference thing. I went with an arrow function to keep things consistent with other similar functions in the code, but I can totally switch to a regular function if that makes more sense!